accepted/2021/workloads/workload-installation.md

```diff
@@ -291,6 +307,7 @@ There will be MSIs for both workload packs and workload manifests.  Both of thes
 ### Workload manifest MSIs
 - MSI should upgrade previous versions of the workload manifest MSI for the same workload manifest ID and SDK Feature Band
   - This is different than most MSIs for the .NET SDK, such as the workload packs, which install side-by-side.
+  - In order to properly support upgrading, the semantic version number for the workload manifest needs to be mapped to a MSI version, preserving the version ordering present in the original semantic version.  As there are only 32 significant bits between the MSI Major, Minor, and Build numbers, it's not possible to do this for arbitrary semantic versions.  The mapping we use should ensure that for all public releases and public previews, the ordering is preserved correctly, and should make a best effort to satisfy this for daily builds.
```

Comment:
This is going to be a tricky one. For example, the current Android manifest package I'm looking at is versioned as 11.2.0-ci.d16-9.0+sha.f8169e5. If possible, having only major.minor versions would be ideal, allowing us to use something like the commit height for the build number.

We should probably also decide on the exact upgrade process. Major upgrades are the easiest to implement and given that the MSI has only a couple of files, we don't have to be concerned with performance. It also allows us to make the most amount of changes between two installers